standard_template/standard/clause_specification_text.adoc

```diff
@@ -294,6 +294,7 @@ Geographic CRSs anchor coordinate values to an ellipsoidal approximation of the
 
  - The value of the `"type"` member MUST be "GeographicCRS"
  - The object MAY have an `"id"` member, whose value MUST be a string and SHOULD be a common identifier for the reference system.
+ - The object MAY have a `"description"` member, whose value must be a string, but no standardised content is interpreted from this description.
```

Comment:
`"description"` is declared throughout the spec and in the JSON-LD context as being an i18n object keyed by language tag. This value type should be used here as well to be consistent.

Reply:
That's a very good point. So I guess the text should say, "... where the value MUST be an i18n object, but no standardised content..."?